kratos/tests/test_dofs.py

```diff
@@ -35,6 +45,20 @@ def test_model_part_sub_model_parts(self):
         self.assertEqual(dx.GetReaction(), KratosMultiphysics.REACTION_X)
         self.assertEqual(dx.EquationId, 5)
 
+        self.assertTrue(p<p2)
+        self.assertTrue(dy>dx)

```

Comment:
There is assertGreater etc :)

Reply:
I left it like this in order to avoid any modification in current code. If this is a problem I've no inconvenience on changing it.

Reply:
It shows up as green thus thought its new

Wont hurt to change it, but up to you

Benefit of the other function is the better error message

Reply:
It is in green because I refactored a bit the previous one to be more unittest-like. In any case I'm changing it, thanks for the suggestion.